docs/source/users/options/fitting_option.rst

```diff
@@ -13,8 +13,8 @@ Software (:code:`software`)
 Software is used to select the fitting software to benchmark, this should be
 a newline-separated list. Available options are:
 
-* ``bumps`` (default software)
-* ``dfo`` (default software)
+* ``bumps`` (external software)
+* ``dfo`` (external software)
 * ``gofit`` (external software -- see :ref:`extra_dependencies`)
 * ``gradient_free`` (default software)
```

Comment:
I think we boh missed this one when talking about it but this is also in the external-instructions bit.